src/macOSNotif.js

```diff
@@ -35,100 +36,100 @@ class __macOSNotifJS_Interact {
         return this;
     }
 
-    scroll_move(evt) {
+    scrollMove(evt) {
         // TODO: make this work, detect X scrolling as a swipe to dismiss
         if (!evt.deltaX) return;
         if (evt.deltaX < 0) this.onDismiss();
     }
 
-    scroll_run() {
-        this.element.addEventListener("wheel", (evt) => this.scroll_move(evt), true);
+    scrollRun() {
+        this.element.addEventListener("wheel", (evt) => this.scrollMove(evt), true);
```

Comment:
I think we should set an eslint rule to enforce no brackets around a single argument for an arrow function.